app/src/main/java/com/concordium/wallet/ui/passphrase/recover/RecoverWalletActivity.kt

```diff
@@ -94,9 +95,15 @@ class RecoverWalletActivity : BaseActivity(), AuthDelegate by AuthDelegateImpl()
         viewModel.saveSeed.observe(this){
             if(it){
```

Comment:
Trying to learn :) What is "it" short for? It's a bit hard to understand what the precondition is for error to be shown.

Reply:
In a lambda expressions "it" is automatically declared based on the expected type. It is used to reduce boilerplate code. For more complex  operations it can be labeled.

Reply:
The lambda is actually a parameter to the `observe` method and it's the [conventional style](https://stackoverflow.com/questions/70213972/why-should-lambda-argument-be-moved-out-of-parentheses) to write it like this. It looks odd to me but I suppose it's just something you get used to. `it` is just a sugar for [single parameter lambdas](https://kotlinlang.org/docs/lambdas.html#it-implicit-name-of-a-single-parameter) that you don't need to explicitly name "it" (I would imagine that they want you to think of it somewhat like `this`).